main.css

```diff
@@ -50,11 +46,11 @@ body {
       font-family: expletus, serif;
       font-size: 4vw;
       text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.7);
-      margin: .5vw 0; }
```

Comment:
Yeah, my bad, not sure how that slipped back given that I thought I pulled the newest version before adding new code. I'm away from my computer, so won't be able to change things till Tuesday.